git_big/main.py

```diff
@@ -861,22 +877,33 @@ def help(ctx, topic, **kw):
 
 @cli.command('version')
 def cmd_version():
-    '''Print version and exit.'''
+    """Print version and exit."""
     click.echo(__version__)
 
 
 @cli.command('init')
 def cmd_init():
-    '''Initialize a repository.'''
+    """Initialize a repository."""
     App().cmd_init()
 
 
+@cli.command('set-depot')
+@click.option('--url', prompt='Depot URL', help='The URL of the git-big depot')
+@click.option('--secret', prompt=True, help='The git-big depot secret')
+@click.option('--key', prompt=True, help='The git-big depot key')
+def cmd_set_depot(url, secret, key):
```

Comment:
It's probably not great for these to show up in people's bash history. Might be better to prompt for these instead.

Reply:
That's what `prompt=` is doing -- I figured this way would make it possible to use without prompting.  Your call, tho.

Reply:
OH! I didn't know click could do that, that's neat!